components/brave_wallet_ui/components/extension/buttons/index.ts

```diff
@@ -0,0 +1,3 @@
+import NavButton from './nav-button'
```

Comment:
Just a question - what's the reason behind the `/extension` part of the path for these? Seems like at least some of these components would apply to all parts of the wallet UI, so would simply go under `components/brave_wallet_ui/components/...`? And also are we actually using an extension at all or just using bundled WebUI?

Reply:
Currently I am building out these "Wallet Extension Panels" from Figma. I agree that some of these components will probably need to be reused throughout the wallet UI. Since the wallet ui is still being designed and not yet finalized, its unapparent at the moment what will be reused. But I will definitely address these components paths as more is built out! :) 

![Screen Shot 2021-03-19 at 9 09 07 AM](https://user-images.githubusercontent.com/40611140/111801752-dbd98300-8892-11eb-901d-b391d9ec4221.png)

Reply:
Oh ok. Maybe we should call the directory "popups" then since it looks like that's what the designer means when they say "extension". It would perhaps be more intuitive for a dev to find where the popup code is since an extension could be a page or a popup etc and maybe we wouldn't even use an extension for the implementation, but a WebUI popup...